script.js

```diff
@@ -1,13 +1,207 @@
-const hamburger = document.querySelector(".hamburger");
-const navMenu = document.querySelector(".nav-menu");
+const hamburger = document.querySelector('.hamburger');
+const navMenu = document.querySelector('.nav-menu');
 
-hamburger.addEventListener("click", () => {
-    hamburger.classList.toggle("active");
-    navMenu.classList.toggle("active");
-    
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navMenu.classList.toggle('active');
 });
 
-document.querySelectorAll(".nav-link").forEach(i => i.addEventListener("click", () => {
-    hamburger.classList.remove("active");
-    navMenu.classList.remove("active");
-}))
\ No newline at end of file
+document.querySelectorAll('.nav-link').forEach((i) => i.addEventListener('click', () => {
+  hamburger.classList.remove('active');
+  navMenu.classList.remove('active');
+}));
+
+// Local Storage
+
+// retrieve the form and details
+
+const formReg = document.getElementById('register');
+const userName = document.getElementById('name');
+const userEmail = document.getElementById('email');
+const userMessage = document.getElementById('message');
+
+formReg.addEventListener('input', () => {
+  // get the values submitted as they are keyd in
+  // store them in js object
+  const data = {
+    name: userName.value,
+    email: userEmail.value,
+    message: userMessage.value,
+  };
+    // stringify the data to store
+
+  const storeData = JSON.stringify(data);
+  localStorage.setItem('data', storeData);
+});
+
+// add event listener function when page is reloaded
+window.addEventListener('load', () => {
+// retrieve inforamtion
+  const userData = JSON.parse(localStorage.getItem('data'));
+  if (userData) {
+    const userName = document.getElementById('name');
+    const userEmail = document.getElementById('email');
+    const userMessage = document.getElementById('message');
+    userName.value = userData.name;
+    userEmail.value = userData.email;
+    userMessage.value = userData.email;
+  }
+});
+
+// Dynamically generate the works section
+
+// store data in js object
+
+const worksData = [
+  {
+    id: 1,
+    Company: 'Tonic',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries',
+    img: '/images/Tonic.svg',
```

Comment:
With this file path, your image is not showing:
<img width="763" alt="image" src="https://user-images.githubusercontent.com/31897434/235469648-ded82978-2999-42ee-8849-3573fa4134ba.png">

Try this instead 👉 `    img: 'images/Tonic.svg',` 

I trust you would correct this before merging 👍